### PR TITLE
Merge two lists

### PR DESCRIPTION
**Pull request by berkus**
This PR proposes to merge changes from the original community [awesome-embedded-rust](https://github.com/berkus/awesome-embedded-rust) list.

It's split in more or less logical commits.

Logo rendering should become normal once it lands in master (or we could remove the logo altogether, however i'd vote for keeping it).

The text changes in the list of crates are made with the assumption than this list is supposed to benefit not only hardcode lowlevel rust devs but mostly newcomers who will use this list to discover new things so more detailed descriptions (even if sometimes rather redundant) will make it easier for them to navigate.

Code of conduct and contibution guidelines files added as per rules of the awesome.re awesome list aggregator.

### CODE REVIEW

**Comment by rahul-thakoor** [comment]
@berkus Wow!

File: README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
-# `awesome-embedded-rust`
+# Embedded Rust
 
-> A curated list of crates useful for embedded development.
+[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```
[the remark this answers is not in the document]
Comment (therealprof) [reply README.md]: Actually I checked with the list and there're two different badges. One if you think you're awesome and one if you're mentioned. So I withdraw my comment. 👍

**Comment by berkus** [comment]
I will address the comments in a separate PR tonight.

File: README.md [hunk 2]
```diff
@@ -34,9 +77,14 @@ Register definition for microcontroller families. Usually generated using [`svd2
...
 
+### ARM
+
+- [`cortex-m`](https://github.com/japaric/cortex-m) Low level access to Cortex-M processors
+
 ### Nordic
 
-- [`nrf51`](https://crates.io/crates/nrf51)
+- [`nrf51`](https://crates.io/crates/nrf51) Peripheral access API for nRF51 microcontrollers (generated using svd2rust)
+- [`nrf51822` playground](https://github.com/japaric/nrf51822) A crate to play with the nrf51822 module
```
[the remark this answers is not in the document]
Comment (berkus) [reply README.md]: ✅

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (berkus) [reply README.md]: ✅

File: README.md [hunk 3]
```diff
@@ -21,8 +30,42 @@
...
+    - [AVR](#avr)
 * [License](#license)
 
+## Community
+
+In 2018 Mozilla asked for creation of an embedded workgroup to help drive adoption in the Rust ecosystem.
+
+- [Embedded WG](https://github.com/rust-lang-nursery/embedded-wg/), including newsletters with progress updates.
+
+- [embedded.rs](https://t.me/embedded_rs) Telegram chat about Rust for microcontrollers in Russian language.
```
[the remark this answers is not in the document]
Comment (berkus) [reply README.md]: ✅

File: Code-of-Conduct.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at berkus@gmail.com. All
```
[the remark this answers is not in the document]
Comment (berkus) [reply Code-of-Conduct.md]: ✅

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Please note that this project is released with a [Contributor Code of Conduct](Code-of-Conduct.md). By participating in this project you agree to abide by its terms.
+
+# The pull request should have a useful title. Pull requests with `Update readme.md` as title will be closed right away because I'm so tired of nobody reading this document. Please carefully read everything in `Adding to this list`.
```
[the remark this answers is not in the document]
Comment (berkus) [reply CONTRIBUTING.md]: ✅

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (berkus) [reply README.md]: I just couldn't find any other official channels except irc channel.

Perhaps there's also a gitter chat? That should be suitable.

**Pull request merged**